src/SamplesApp/UITests.Shared/Microsoft_UI_Xaml_Controls/TreeView/TreeViewPage.xaml.cs

```diff
@@ -0,0 +1,1010 @@
+using TreeViewList = Microsoft.UI.Xaml.Controls.TreeViewList;
+using TreeViewItem = Microsoft.UI.Xaml.Controls.TreeViewItem;
+using System.Threading.Tasks;
+using Uno.UI.Samples.Controls;
+using MUXControlsTestApp.Utilities;
+using MUX.UITests.Shared.Microsoft_UI_Xaml_Controls.TreeViewTests;
+
+namespace UITests.Shared.Microsoft_UI_Xaml_Controls.TreeViewTests
+{
+	[SampleControlInfo("TreeView", "MUX_Test_TreeViewPage")]
+	public sealed partial class TreeViewPage : MUXTestPage
+	{
+		bool _disableClickToExpand;
+		public TreeViewItem flyoutTVI;
+		TreeViewNode _visualRoot;
+		TreeViewNode _virtualizedNode;
+		public ObservableCollection<TreeViewItemSource> TestTreeViewItemsSource { get; set; }
+		public ObservableCollection<TreeViewItemSource> TestTreeView2ItemsSource { get; set; }
+
+		public TreeViewPage()
+		{
+			this.DataContext = this;
+			this.InitializeComponent();
+
+			//TODO
+			//MaterialHelperTestApi.IgnoreAreEffectsFast = true;
+			//MaterialHelperTestApi.SimulateDisabledByPolicy = false;
+
+			_disableClickToExpand = false;
+
+			SetupCustomDragUIOverride.Click += SetupCustomDragUIOverride_Click;
+
+			_visualRoot = CreateTree();
+			TestTreeView.RootNodes.Add(_visualRoot);
+
+			TestTreeView2.RootNodes.Add(new TreeViewNode() { Content = "item1" });
+			TestTreeView2.RootNodes.Add(new TreeViewNode() { Content = "item2" });
+
+			TestTreeViewItemsSource = PrepareItemsSource();
+
+			var item1 = new TreeViewItemSource() { Content = "item1" };
+			var item2 = new TreeViewItemSource() { Content = "item2" };
+			TestTreeView2ItemsSource = new ObservableCollection<TreeViewItemSource>() { item1, item2 };
+		}
+
+		private ObservableCollection<TreeViewItemSource> PrepareItemsSource(bool expandRootNode = false)
+		{
+			var root0 = new TreeViewItemSource() { Content = "Root.0" };
+			var root1 = new TreeViewItemSource() { Content = "Root.1" };
+			var root2 = new TreeViewItemSource() { Content = "Root.2" };
+			var root = new TreeViewItemSource() { Content = "Root", Children = { root0, root1, root2 }, IsExpanded = expandRootNode };
+
+			return new ObservableCollection<TreeViewItemSource> { root };
+		}
+
+		private async Task<ObservableCollection<TreeViewItemSource>> PrepareItemsSourceAsync()
+		{
+			return await Task.Run(() => {
+				var items = PrepareItemsSource(expandRootNode: true);
+
+				var root0 = items[0].Children[0];
+				var x0 = new TreeViewItemSource { Content = "Root.0.0" };
+				root0.Children.Add(x0);
+
+				var root1 = items[0].Children[1];
+				var y0 = new TreeViewItemSource { Content = "Root.1.0" };
+				var y1 = new TreeViewItemSource { Content = "Root.1.1" };
+				root1.Children.Add(y0);
+				root1.Children.Add(y1);
+				root1.IsExpanded = true;
+
+				return items;
+			});
+		}
+
+		protected internal override void OnNavigatedFrom(NavigationEventArgs e)
+		{
+			// Unset all override flags to avoid impacting subsequent tests
+			//TODO
+			//MaterialHelperTestApi.IgnoreAreEffectsFast = false;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5156819)